### ISSUE TITLE
Training

### ISSUE THREAD

**Issue by jsczzzk** [sniklaus/pytorch-liteflownet#2]
Thank you for your outstanding work!

Will training be available in the future?

**Comment by sniklaus** [comment]
I am afraid that my time is limited and I do not have the resources to provide the training code. The provided implementation is fully differentiable though, which is why you just need to provide a data loader and implement a loss function. Feel free to fork this repository and see whether you can make it work. It would be nice to have a fork with a working training section that I could link to.
